src/store/UserStore.ts

```diff
@@ -1,19 +1,52 @@
 import CookieUtils from 'js-cookie';
 
-interface User {
-    upn: string;
-    displayName: string;
-    isRamad: boolean;
-}
+import { observable, decorate, action, computed } from 'mobx';
+
+import { unsetFavoriteAd, setFavoriteAd } from '../server/user';
+import { User } from '../types/User';
 
 export class UserStore {
-    // Get the user info via cookie
-    private loggedUser = JSON.parse(CookieUtils.get('user') || '{}');    
-    
-    get user(): User {
-        return this.loggedUser;
+    private loggedUser = observable.box<User>(JSON.parse(CookieUtils.get('user') || '{}'));
```

Comment:
I didn't think we should make it observable sience it's static...
We could just provide it through the store and make it a property

Reply:
Done